### PR TITLE
Add changing Tapestry settings 

### LINKED ISSUE #24: Add pop-up for changing Tapestry settings (admin only)

### PR DESCRIPTION
**Pull request by anhduc130**
Fixes #24

### CODE REVIEW

File: wp-content/plugins/tapestry/controller/class.tapestry-controller.php [hunk 1]
```diff
@@ -55,6 +55,26 @@ public function updateTapestry($tapestry) {
...
+     * @param  Object @settings
+     * @return Object Tapestry settings
+     */
+    public function updateTapestrySettings($settings) {
+        if (!$this->postId) {
+            return $this->_throwsError('INVALID_POST_ID');
+        }
+
+        $tapestry = get_post_meta($this->postId, 'tapestry', true);
+        $tapestry->settings = $settings;
```
Comment (wynnset) [inline wp-content/plugins/tapestry/controller/class.tapestry-controller.php]: Could you add a todo here to add some validation for the $settings that is being passed in?

File: wp-content/plugins/tapestry/controller/class.tapestry-controller.php [hunk 2]
```diff
@@ -55,6 +55,26 @@ public function updateTapestry($tapestry) {
...
+
+        $this->_updatePost($tapestry, 'tapestry');
```
Comment (wynnset) [inline wp-content/plugins/tapestry/controller/class.tapestry-controller.php]: Is this doing anything? If not, could you please remove it? Thanks.

Comment (anhduc130) [reply wp-content/plugins/tapestry/controller/class.tapestry-controller.php]: Yes, it is used to update the post title of the post.

File: wp-content/plugins/tapestry/controller/class.tapestry-controller.php (hunk 1 above)
Comment (anhduc130) [reply wp-content/plugins/tapestry/controller/class.tapestry-controller.php]: Should it be validated in the `endpoints.php`?

File: wp-content/plugins/tapestry/controller/class.tapestry-controller.php (hunk 2 above)
Comment (wynnset) [reply wp-content/plugins/tapestry/controller/class.tapestry-controller.php]: Are you sure? The body of `_updatePost` function is setting the title of the post to `$post->title`, not `$post->settings->title` so either that function is wrong or this shouldn't be here.

Comment (wynnset) [reply wp-content/plugins/tapestry/controller/class.tapestry-controller.php]: Oh sorry, I was looking at `TAPESTRY_NODE` .. I see that for `TAPESTRY` it is using settings.

File: wp-content/plugins/tapestry/controller/class.tapestry-controller.php (hunk 1 above)
Comment (wynnset) [reply wp-content/plugins/tapestry/controller/class.tapestry-controller.php]: I think doing the validation here makes more sense actually.

**Pull request merged**